package.json

```diff
@@ -60,11 +55,5 @@
   },
   "browserslist": [
     "last 2 versions"
-  ],
-  "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "prettier --write"
-    ]
-  }
```

Comment:
You can tell it's not used:
1. it targets non-existent JS files
2. It uses `eslint`, which is not installed